### ISSUE TITLE
OBS can't capture sandboxed games

### ISSUE THREAD

**Issue by ghost** [sandboxie-plus/Sandboxie#2945]
### Describe what you noticed and did

1. Launch OBS (Open Broadcaster Software) outside Sandboxie
2. Add a source (Mode: Capture any fullscreen application OR Capture specific window)
3. Launch a game from one of boxes (tested both Standard and Application Compartment with the same result)
4. Once the game is launched wait a few seconds and then alt+tab back to OBS to see if the game is seen in the preview window
5. The preview window always displays black
6. In-game recorded videos and in-game taken screenshots are always black

So far I've tested it with Unreal Gold, Unreal Tournament, Arma: Cold War Assault and Söldner: Secret Wars. The same games launched outside Sandboxie are captured correctly by OBS. Standard Applications of a box such as sandboxed Windows Explorer and Registry Editor can be captured by OBS correctly.
Tested with "Ask for UAC Elevation" and "Emulate Admin Rights" Sandboxie presets.

### How often did you encounter it so far?

Always

### Affected program

Open Broadcaster Software 27.2.4 (64-bit)

### Download link

https://obsproject.com/

### Where is the program located?

The program is installed only outside the sandbox.

### Expected behavior

OBS can capture sandboxed games.

### What is your Windows edition and version?

Windows 7 x64

### In which Windows account you have this problem?

An account with UAC protection completely turned off.

### Please mention any installed security software

Microsoft Security Essentials

### What version of Sandboxie are you running?

Sandboxie Plus 1.9.3 64-bit

### Is it a new installation of Sandboxie?

I have been using the same version for some time.

### In which sandbox type you have this problem?

All sandbox types (I tried them all).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

**Comment by isaak654** [comment]
#2757

**Comment by bastik-1001** [comment]
Does OBS fail to capture anything, running under Sandboxie, like the browser window or the explorer window or is it "just" that games won't be captured? I just ask, even though the title mentions games.

There are feature requests to block screen recordings, so whenever this is applicable to every application, any required change can be used to get closer to fulfill those requests.

**Comment by DavidXanatos** [comment]
can you find some small free game for which the is fails so I can test with?

**Comment by isaak654** [comment]
#3374

**Comment by 769687520** [comment]
> 作为问题的补充，事实证明，当两者从同一个沙盒启动时，沙盒 OBS 可以捕获在视图模式下启动的游戏。全屏启动的同一游戏无法被 OBS 捕获。 操作系统中的OBS（沙盒外）无论是窗口还是全屏模式都无法捕获沙盒游戏。

Indeed, I have also noticed this issue. If you run it all inside the sandbox, you can catch it, but not outside the sandbox!
